spec/plugin.spec.js

```diff
@@ -221,4 +221,48 @@ describe('ManifestPlugin', function() {
       });
     });
   });
+
+  describe('assets added by third parties to "compilation.assets"', function() {
+    it('are included also when enabled.', function(done) {
+
+      var options = webpackConfig({
+        manifestOptions: {
+            includeCopiedAssets: true
+        }
+      });
+
+      // add plugin to get third party assets included
+      let Mock = require(path.join(__dirname, './plugin-mock'));
+      options.plugins.unshift(new Mock());
+
+      webpackCompile(options, function(manifest, stats) {
+        expect(manifest['third.party.js']).toEqual('third.party.js');
+        done();
+      });
+    });
+  });
+
+  describe('assets added by third parties to "compilation.assets"', function() {
```

Comment:
I would like to have a way to test it without using the plugin. Any idea how best to do. Currently I always add it.